Plugin/EnvironmentPlugin/ProjectDescriptionHelpers/ProjectEnvironment.swift

```diff
@@ -6,14 +6,20 @@ public struct ProjectEnvironment {
     public let name: String
     public let organizationName: String
     public let deploymentTarget: DeploymentTarget
```

Comment:
```suggestion
    @available(*, deprecated, message: "'DeploymentTarget' was deprecated, use instead of 'DeploymentTargets' and 'Destinations'")
    public let deploymentTarget: DeploymentTarget
```
deprecated 달아놓는거 어떤가용?